librfn/libopencm3/console_cdcacm.c

```diff
@@ -233,28 +233,26 @@ static struct output_task output_task = {
 #if defined(STM32F1)
 static void usb_hwinit(void)
 {
-	uint32_t t;
-
 	rcc_periph_clock_enable(RCC_GPIOA);
-	rcc_periph_clock_enable(RCC_GPIOB);
+	rcc_periph_clock_enable(RCC_AFIO);
```

Comment:
IIRC then removing GPIOB caused problems for me. Are you turning this clock on somewhere else in your test application?